### DISCUSSION TITLE
axios version 1.x.x not correctly working with react testing library

### DISCUSSION THREAD

**Discussion by seyed-amir-mehrizi** [axios/axios#5230] [Q&A]
Hi,
I am using axios version ^1.1.3 with react testing library to test the react app. After updating axios to the new version from ^0.24.0 to the latest one, After running my test I get this error : 

 ({"Object.<anonymous>":function(module,exports,require,__dirname,__filename,jest){import axios from './lib/axios.js';

I searched a lot on the net but did not find any issues. But when I back to version ^0.24.0, All my tests pass.
I hope someone can help me with new version.

**Comment by dredav** [comment]
https://github.com/axios/axios/pull/5162
